terraform/projects/pr-dashboard/rds.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_db_subnet_group" "rds" {       
+  name = "default-db"                        
+  description = "default subnets"            
+  subnet_ids = ["${aws_subnet_wpt_id}"]
+}
+
+resource "aws_subnet_wpt" "wpt-subnet" {
+  vpc_id            = "vpc-eb65f992"
+  cidr_block        = "10.100.0.0/16"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_db_instance" "rds" {
+  identifier = "web-platform-tests-db"
+  storage_type = "gp2"
+  allocated_storage = 10
+  engine = "postgres"
+  engine_version = "9.5.4"
+  instance_class = "db.t2.small"
+  name = "wptdash"
+  username = "wptdash"
+  password = "${trimspace(file("${path.module}/secrets/wpt-db-password.txt"))}"
```

Comment:
Note, you'll need this file in place, and it doesn't exist in the repo right now as its public.

Reply:
Roger